spec/vast_verification.spec.js

```diff
@@ -0,0 +1,122 @@
+import { parserVerification } from '../src/parser/parser_verification';
+import { vastInLine } from './samples/inline_missing_values';
+import { vastMissingWrapperValues } from './samples/wrapper_missing_values';
+
+describe('parserVerification', function() {
```

Comment:
Right now you tested that you get warnings for all the things you needed, but you didn't add any tests to check that `verifyRequiredValues`, `verifyRequiredAttributes `, `verifyRequiredSubElements `, `hasSubElements `, `emitMissingValueWarning ` individually do what you expect them to do.